### ISSUE TITLE
Wiremock 3.0.0 404 error

### ISSUE THREAD

**Issue by thomasam000** [modernweb-pl/wiremock-standalone#21]
On version 3.1.2

when running install.js, I receieve the error

```
Downloading WireMock standalone from Maven Central...
 https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-jre8-standalone/3.0.1/wiremock-jre8-standalone-3.0.1.jar
 Error: Request failed with status code 404
```

Wiremock 3.0.0 came out yesterday, and it appears that wiremock 3 doesn't have a jar file.
https://repo1.maven.org/maven2/com/github/tomakehurst/wiremock-jre8-standalone/3.0.1/

perhaps you can use the latest version of wiremock 2

**Comment by thomasam000** [comment]
https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/3.0.0/wiremock-standalone-3.0.0.jar

this is a working wiremock 3 link, not sure if this would work

**Comment by frederic-kneier** [comment]
I already created a PR for this problem: https://github.com/modernweb-pl/wiremock-standalone/pull/22

**Comment by ani-shk** [comment]
Would love to have this fixed asap :)

**Comment by Hagith** [comment]
#22 released in v3.2.0 and published to [NPM](https://www.npmjs.com/package/wiremock-standalone/v/3.2.0)

**Comment by Hagith** [comment]
Also fixed that wiremock >= 3 has no separate JRE versions, published as v3.2.1.
